src/components/contextual/pages/pool/PoolStatCards.vue

```diff
@@ -10,7 +10,7 @@
         </div>
         <div class="text-xl font-medium truncate flex items-center">
           {{ stat.value }}
-          <LiquidityMiningTooltip :pool="pool" v-if="stat.id === 'apr'" />
+          <LiquidityAPRTooltip :pool="pool" v-if="stat.id === 'apr'" />
```

Comment:
Its not just about LM now, so the tooltip name changed to something more generic.